lfs/ntlm.go

```diff
@@ -0,0 +1,172 @@
+package lfs
+
+import (
+	"bytes"
+	"encoding/base64"
+	"github.com/ThomsonReutersEikon/go-ntlm/ntlm"	
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+func (c *Configuration) NTLMSession(creds Creds) ntlm.ClientSession {
+	
+	if c.ntlmSession != nil {
+		return c.ntlmSession
+	}
+	
+	splits := strings.Split(creds["username"], "\\")
+	var session, _  = ntlm.CreateClientSession(ntlm.Version2, ntlm.ConnectionOrientedMode)
```

Comment:
Looks like this is hiding a possible error. I think that this function should be private, and should return a possible error that other functions can bubble up.